pkg/utils/databackup.go

```diff
@@ -83,3 +85,27 @@ func ParseBackupRestorePath(backupRestorePath string) (pvcName string, path stri
 	}
 	return
 }
+
+// CalTimeCost calculate the time cost of DataBackup
+// the result is a string in the form of **h**m**s
+func CalTimeCost(startTime int64) (timeCost string) {
```

Comment:
I suggest to use 'time' package. https://golang.org/pkg/time/

I think there are some references in K8s implemtation.